### DISCUSSION TITLE
release date for 4.3 or 5.0?

### DISCUSSION THREAD

**Discussion by jbloom** [vega/altair#2588] [General]
I've been watching the development of these new versions. I was curious if there is an anticipated timeline for their release, mostly because I'm starting a project that uses `altair` a lot and trying to figure out if I should just wait for 5.0 or get started now.

**Comment by joelostblom** [comment]
I don't think there is any target date for the next release. I believe most of the new functionality is already in the current GitHub version + some of the open PRs ready to be merged, but there are a few cases still being worked on, maybe most notable https://github.com/altair-viz/altair/issues/2573 but also https://github.com/altair-viz/altair/issues/2584, https://github.com/altair-viz/altair/pull/2568 and others.

So it is likely quite safe to use the Altair development version and expect a pretty seemless upgrade once the next version rolls out, but at the same time there is no guarantee that this will be the case. I use it myself, but I am accepting the risk of redoing things if something breaks before release.

**Reply by saiwing-yeung** [reply]
Wondering if we can bump the version to 4.3? I use `conda/mamba` for package management and prefer not to get the GitHub version directly. I'd really appreciate that.

**Reply by joelostblom** [reply]
That would be nice, but it would require some reordering of the commits on the main branch to make sure only the small fixes are included in a 4.3 update and all the major functionality is not included until a 5.0 release. I am not sure if anyone one the current dev team has the time to take that on and make a release, all of our efforts and time are focused o the next major release right now.

**Reply by saiwing-yeung** [reply]
No worries. Thanks for the great work. Looking forward to 5.0!

**Reply by joelostblom** [reply]
@saiwing-yeung @jbloom There is now a release candidate of Altair 5, which supports Vega-Lite 5! Details and info on how to install can be found  in https://github.com/altair-viz/altair/issues/2937

**Reply by saiwing-yeung** [reply]
@joelostblom Thanks for the heads up and look forward to v5~
